woocommerce-admin-test-helper.php

```diff
@@ -54,7 +57,33 @@ function add_extension_register_script() {
 	wp_enqueue_style( 'woocommerce-admin-test-helper' );
 }
 
-add_action( 'admin_enqueue_scripts', 'add_extension_register_script' );
+add_action( 'admin_enqueue_scripts', 'add_wc_admin_helper_extension_register_script' );
+
+function add_extended_task_list_script() {
+	$inline_config = array(
+		'extendedTasks' => json_decode(get_option('woocommerce_admin_test_helper_extended_tasks', '[]'))
+	);
+	wp_register_script( 'woocommerce-admin-test-helper-config', '' );
+	wp_enqueue_script( 'woocommerce-admin-test-helper-config' );
+	wp_add_inline_script( 'woocommerce-admin-test-helper-config', 'window.wcAdminTestHelperConfig = ' . wp_json_encode( $inline_config ), 'before' );
+
+	$script_path       = '/build/extended-tasks.js';
+	$script_url = plugins_url( $script_path, __FILE__ );
+	$script_asset_path = dirname( __FILE__ ) . '/build/extended-tasks.asset.php';
+	$script_asset      = file_exists( $script_asset_path )
+		? require( $script_asset_path )
+		: array( 'dependencies' => array(), 'version' => filemtime( $script_path ) );
+
+	wp_enqueue_script(
+		'woocommerce-admin-test-helper-extended-tasks',
+		$script_url,
+		$script_asset['dependencies'],
+		$script_asset['version'],
+		true
+	);
+}
+
+add_action( 'admin_enqueue_scripts', 'add_extended_task_list_script' );
```

Comment:
This loads the task-list filter logic separately on any admin page, although only used on the **WooCommerce > Home** page.
It loads the inline `wcAdminTestHelperConfig` and the `extended-tasks.js` file, bundled separately as shown in the webpack.config